package.json

```diff
@@ -55,27 +55,26 @@
     "cssnano": "3.10.0",
     "fs-extra": "1.0.0",
     "git-rev-sync": "1.4.0",
-    "grunt": "0.4.5",
-    "grunt-cli": "0.1.13",
-    "grunt-contrib-connect": "0.11.2",
-    "grunt-contrib-copy": "0.8.0",
+    "grunt": "^1.0.3",
+    "grunt-cli": "^1.3.1",
+    "grunt-contrib-connect": "^2.0.0",
+    "grunt-contrib-copy": "^1.0.0",
     "grunt-contrib-jasmine": "1.1.0",
-    "grunt-contrib-jshint": "0.11.2",
-    "grunt-contrib-requirejs": "0.4.4",
-    "grunt-contrib-watch": "1.0.0",
+    "grunt-contrib-watch": "^1.1.0",
     "grunt-eslint": "^19.0.0",
-    "grunt-exec": "0.4.6",
-    "grunt-json-generator": "0.1.0",
-    "grunt-postcss": "0.8.0",
-    "grunt-retire": "0.3.12",
-    "grunt-sass": "2.0.0",
+    "grunt-exec": "^3.0.0",
+    "grunt-json-generator": "^0.1.0",
+    "grunt-postcss": "^0.9.0",
+    "grunt-retire": "^1.0.7",
+    "grunt-sass": "^3.0.1",
```

Comment:
Given the scope of fixing outdated  dependencies + `npm audit` warnings, I think this change makes a lot of sense.

One thing we should consider prioritizing is moving from grunt to webpack. We should be able to use the [`sass-loader`](https://github.com/webpack-contrib/sass-loader) instead.